<ui version="4.0" >
 <class>AAddUserDialogClass</class>
 <widget class="QDialog" name="AAddUserDialogClass" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>User Dialog</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="aforms.qrc" >:/images/mainabase.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="upUserDataBox" >
     <property name="title" >
      <string>User Data</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="upUsernameLabel" >
        <property name="text" >
         <string>Username:</string>
        </property>
        <property name="buddy" >
         <cstring>upUsernameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="upUsernameLineEdit" >
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="upUsernameErrorLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QCheckBox" name="upChgPasswordCheckBox" >
        <property name="text" >
         <string>Change Password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="upPasswordLabel" >
        <property name="text" >
         <string>Password:</string>
        </property>
        <property name="buddy" >
         <cstring>upPasswordLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="upPasswordLineEdit" >
        <property name="maxLength" >
         <number>20</number>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="upConfirmPasswordLabel" >
        <property name="text" >
         <string>Confirm Password:</string>
        </property>
        <property name="buddy" >
         <cstring>upConfirmLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="upConfirmLineEdit" >
        <property name="maxLength" >
         <number>20</number>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <widget class="QLabel" name="upPasswordErrorLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="upSecurityLevelLabel" >
        <property name="text" >
         <string>Security Level:</string>
        </property>
        <property name="buddy" >
         <cstring>upSecurityComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QComboBox" name="upSecurityComboBox" />
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="upEndDateLabel" >
        <property name="text" >
         <string>End Date:</string>
        </property>
        <property name="buddy" >
         <cstring>upEndDate</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1" >
       <widget class="QDateEdit" name="upEndDate" >
        <property name="calendarPopup" >
         <bool>true</bool>
        </property>
        <property name="date" >
         <date>
          <year>2008</year>
          <month>1</month>
          <day>16</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="8" column="0" >
       <widget class="QLabel" name="upCommentLabel" >
        <property name="text" >
         <string>Comment:</string>
        </property>
        <property name="buddy" >
         <cstring>upCommentLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1" >
       <widget class="QLineEdit" name="upCommentLineEdit" >
        <property name="maxLength" >
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upOkButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="aforms.qrc" />
 </resources>
 <connections/>
</ui>
